use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/solver/justifications/MinimizeTravelTimeJustification.java

```diff
@@ -0,0 +1,15 @@
+package org.acme.vehiclerouting.solver.justifications;
+
+import ai.timefold.solver.core.api.score.stream.ConstraintJustification;
+
+import java.time.Duration;
+
+public record MinimizeTravelTimeJustification(String vehicleName, long totalDrivingTimeSeconds,
+        String description) implements ConstraintJustification {
+
+    public MinimizeTravelTimeJustification(String vehicleName, long totalDrivingTimeSeconds) {
+        this(vehicleName, totalDrivingTimeSeconds, "Vehicle '%s' total travel time is %d hours %d minutes."
+                .formatted(vehicleName, Duration.ofSeconds(totalDrivingTimeSeconds).toHours(),
+                        Duration.ofSeconds(totalDrivingTimeSeconds).toMinutesPart()));
```

Comment:
Nitpick: if `toSecondsPart() > 0` add one minute.

Reply:
Done - in fact, I used `toSecondsPart() >= 30` to round it more properly (and have it aligned with the UI, the values were different which seemed weird).